FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.0.1
+- [fixed] Added app.google (1p domain) support in FDL SDK which was missing.
```

Comment:
Include in 10.0.0 which won't code freeze until next week.

Reply:
Done